geiger/src/util/Collection.js

```diff
@@ -0,0 +1,144 @@
+import {isArray} from 'util';
+import defaults from 'lodash.defaults';
```

Comment:
Uh-oh. Do we have a dependency on `lodash`?

Reply:
Just one function called `defaults`. I can rewrite it.